lib/ansible/modules/cloud/amazon/cloudformation.py

```diff
@@ -1,18 +1,9 @@
 #!/usr/bin/python
 # This file is part of Ansible
```

Comment:
I like to get rid of this line (it's part of the long-form GPL recommendation and we're switching to the one-liner).

I also like to add a Copyright notice.  That can require trawling through history which can be time consuming so I have been doing:
``` python
# Copyright: (c) Ansible Project
```

Which is okay for our project.  (If anyone needs to do something legal-related, like relicense, we would have to trawl through git history no matter what the Copyright header says.)  It leaves out the copyright year which I dislike doing but the git history for modules can be time consuming to look through to find the last "copyrightable" change.  I figure if someone complains we can trawl through git history at that time.

For this particular module, we're lucky.  in git log -p I see that there were some copyrightable changes this year so the copyright line can be:
``` python
# Copyright: (c) 2017, Ansible Project
```